.github/workflows/deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - development
```

Comment:
We don't have a dev lambda bucket, so I'd say we should just push once, to prod.